src/strings/spanish.js

```diff
@@ -73,6 +73,8 @@ const SPANISH: TranslateStringsType = {
   Home: 'Hogar',
   FAQ: 'FAQ',
   beforeStart: 'Antes de comenzar...',
+  legalInformationNotAdvice: '',
```

Comment:
@bitcode Can you help with the translations here? I don't think that `useThisForm` is the complete translation.  We also broke it up into separate parts so that the **Does not track** disclaimer is more obvious.

Reply:
I'll be glad to, I'm looking into it, sorry for the late reply